### DISCUSSION TITLE
DDL not working with qbittorrent

### DISCUSSION THREAD

**Discussion by ujjwalkirorimal** [anasty17/mirror-leech-telegram-bot#1215] [General]
**Describe the bug**
Direct downloading links aren't working with qbittorrent

**To Reproduce**
1. /qbmirror https://speed.hetzner.de/100MB.bin
2. Telegram message says " invalid token:b'\x00' "
3.  Bot hangs up and ram usage goes bonkers.

* Everything working fine with torrent cloning and aria2c 

Logs - https://katb.in/uciqitajofi
Idle server stats - https://katb.in/raliqejomep

**Comment by anasty17** [comment]
![20221115_165731](https://user-images.githubusercontent.com/77075674/201951448-6bb2a6ce-c5d3-4eee-9136-5702bea06045.jpg)
READ

**Comment by anasty17** [comment]
How u are using this code without minimum knowledge about qbittorrent! Don't deploy and don't open any other issue in this repository.

**Reply by ujjwalkirorimal** [reply]
> How u are using this code without minimum knowledge about qbittorrent! Don't deploy and don't open any other issue in this repository.

Ah lol man it just skipped my mind that qbitorrent wont work with DDL. I just panicked when bot stopped responding after command. My bad
